CONTRIBUTING.md

```diff
@@ -1,54 +1,59 @@
 # CONTRIBUTING
-
-Contributing to the official buyTogether Application
+Contributing to the official buyTogether application
 Welcome! We're so excited that you're considering contributing to the official GitHub page for buyTogether.
-
 ---
-
-## Make changes locally
-
-- Fork the repo so that you can make your changes without affecting the original project until you're ready to merge them.
-
-- Create a working branch 
-
-- Set up a development environment to begin working on the project all you need to do is 'vagrant up'
-
+## Prerequisites
+- Fork the repo to make your changes without affecting the original project until you're ready to merge.
+- Create a working branch.
+- Set up a development environment to begin working on the project, all you need to do is 'vagrant up'
```

Comment:
Hey, I'm a new developer who wish to contribute to this project! I ran `vagrant up` like you suggested, and got `bash: vagrant: command not found`, any idea what the problem it? :)

Reply:
installing vagrant, right...✔